contracts/bond/CollateralWhitelist.sol

```diff
@@ -3,38 +3,40 @@ pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/IERC20MetadataUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/structs/EnumerableSetUpgradeable.sol";
+
+import "hardhat/console.sol";
```

Comment:
issue (blocker): no hard console in production code 😁 

_I know you're working on this PR, probably best to set it to a draft PR ...otherwise someone may approve & merge in 😉 _